<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyGpio</class>
 <widget class="QWidget" name="MyGpio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>291</width>
     <height>211</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_O_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>IO_OUT_1_L</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_O_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>IO_OUT_2_L</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_O_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>IO_OUT_3_L</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_O_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>IO_OUT_4_L</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_I_1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>IO_IN_1_L</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_I_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>IO_IN_2_L</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_I_3">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>IO_IN_3_L</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_I_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>150</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>IO_IN_4_L</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>4</width>
     <height>13</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
